libs/blocks/quiz/quiz.css

```diff
@@ -11,7 +11,7 @@
 }
 
 .quiz {
-  min-height: calc(100vh - 131px); /* window height minus global nav + global footer heights */
+  min-height: calc(100vh - 64px); /* window height minus global nav height */
```

Comment:
Do we have a var for this?
Also the global navigation can be either with or without breadcrumbs, which changes the height

Reply:
There are actually some variables that could be used, defined here - https://github.com/adobecom/milo/blob/main/libs/styles/styles.css#L4-L6

Reply:
Thanks for the pointer to styles.css. I imported it take advantage of the variable you mentioned. The quiz designs always request an absolutely minimal header, so I believe we're on with the assumption we're making about the height, especially not that we can take advantage of that shared variable.